packages/ozone/src/db/schema/verification.ts

```diff
@@ -0,0 +1,18 @@
+export const verificationTableName = 'verification'
+
+export interface Verification {
+  uri: string
+  issuer: string
+  subject: string
+  handle: string
+  displayName: string
+  revokeReason: string | null
+  revokedBy: string | null
+  revokedAt: string | null
+  createdAt: string
+  updatedAt?: string
```

Comment:
We generally don't want to mark fields in the model as optional, since they always are present when reading data.  If you're looking for the convenience of not including the field on writes, you may want to use kysely's `Generated` type helper.